AnnoDesigner/Localization/Localization.cs

```diff
@@ -211,6 +211,8 @@ public static void Init(ICommons commons)
                         { "ColorTypeCustom" , "Custom" },
                         { "Warning" , "Warning" },
                         { "FileNotFound" , "The file was not found." },
+                        { "ItemsCopied" , " items copied" },
```

Comment:
We shouldn't include leading or trailing spaces within localization information. These spaces should be added wherever the localized value is used.

Reply:
I also adjusted the spreadsheet.